### DISCUSSION TITLE
1.0.37.1

### DISCUSSION THREAD

**Discussion by sylikc** [sylikc/jpegview#2] [zzArchive]
[1.0.37.1]
Other changes:
- First release from sylikc's GitHub
- Using Visual Studio 2019 Community Edition to compile
- Updated to Turbo JPEG version 2.1.0 (4/23/2021)
- Updated to Google's WebP library version 1.2.0 HEAD at commit 05b72d4 (4/28/2021)

<hr /><em>This discussion was created from the release <a href='https://github.com/sylikc/jpegview/releases/tag/v1.0.37.1'>1.0.37.1</a>.</em>

**Comment by Fzygs** [comment]
love this tiny tool ! perfect for me ,thanks for ur work ...

**Comment by bitsper2nd** [comment]
Thanks for keeping the project alive. This is the fastest image viewer I have used on Windows. I am not a programmer, so I can't help you with code, but I am a designer. In case you are looking to improve the look JPEGView, I can help you with that.

**Comment by curvedlinee** [comment]
i wish to have more command line options or for JPEGView.ini to have IDM_ZOOM_200
JPEGView.exe filename /ini myzoom2x.ini or  /IDM_ZOOM_200  or /zoom=1.3x
i wish to have more options for
IDM_ZOOM_230
IDM_ZOOM_190
IDM_ZOOM_180
IDM_ZOOM_170
IDM_ZOOM_130
IDM_ZOOM_120

thanks for making ZoomPausePercent=0
i integrated jpegview with ZTree, which basically is command line utility  with tens of 1000 context menu entries since 40 years old XTree
